MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -410,7 +410,7 @@ protected void setPlaybackHeadUpdateInterval() {
       updatePlayheadPositionTimer.cancel();
     }
     if (playItemHaveVideoTrack) {
-      Player.VideoComponent videoComponent = player.get().getVideoComponent();
+      ExoPlayer.VideoComponent videoComponent = player.get().getVideoComponent();
```

Comment:
Does this cause any backwards compatibility issues? It would likely be compile time that's an issue, but just to be safe I wanted to ask.

Reply:
I think not, as you mentioned it would fail to compile